### PR TITLE
temporarily disable devtoolset7 nightlies

### PR DESCRIPTION
**Pull request by kostmo**
toward issue #20066

### CODE REVIEW

**Comment by pjh5** [comment]
Can we only disable the cuda9.0 devtoolset binaries? All the others are working fine

File: .circleci/cimodel/data/binary_build_data.py [hunk 1]
```diff
@@ -60,7 +60,10 @@ def get_processor_arch_name(cuda_version):
 )
 
 
-DEVTOOLSET_VERSIONS = [3, 7]
+DEVTOOLSET_VERSIONS = [
+    3,
+#    7,  # XXX Temporarily disabled---see https://github.com/pytorch/pytorch/issues/20066
```
Comment (pjh5) [inline .circleci/cimodel/data/binary_build_data.py]: Can we only remove the cuda 9.0 devtoolset binaries? The others are working fine.

**Review by pjh5 — CHANGES_REQUESTED** [review]

**Comment by kostmo** [comment]
# Before:
before-binarybuilds-config-dimensions

# After:
after-binarybuilds-config-dimensions

**Review by pjh5 — APPROVED** [review]
Thanks!

**Pull request closed without merging**